config/webpack.config.base.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
       // JavaScript / ES6
       {
         test: /\.jsx?$/,
-        include: /src\/client\/assets\/javascripts/,
+        include: path.resolve(__dirname, "../src/client/assets/javascripts"),
```

Comment:
Can you please change quotes to be single quotes, since that's what i use in my codebase for strings?